google/earthenterprise
# WIP: Initial code for portable WMS

This adds initial support for WMS to portable. The changes include the following:

- The existing WMS code is included in the portable bundle
- A WMS endpoint is added to the portable server
- The existing WMS code is called from the WMS endpoint
- Removes the unused `pywms` folder

Note that this PR only includes initial support for WMS - the portable server will respond to WMS requests but it may only return errors. Future PRs will fix the WMS code to work fully with portable.

To test these changes:

1. Build, install, and run portable using the instructions here: https://github.com/google/earthenterprise/wiki/Portable-Server
2. In a browser, go to http://localhost:9335/wms/anything. It should return an error message indicating that an invalid WMS request was received.
3. If you submit a valid WMS query like http://localhost:9335/wms?version=1.3.0&request=GetCapabilities&service=WMS you should get a valid WMS response but it will likely indicate there was an error.
4. Since changes were made to the WMS code, you should also run a basic WMS test using the standard server.

This code partially completes issue #869
Fixes #118
